OpenRA.Game/Graphics/CursorSequence.cs

```diff
@@ -1,4 +1,4 @@
-#region Copyright & License Information
+﻿#region Copyright & License Information
```

Comment:
Please uncommit this and remember not to include this in the future.
We don't want to reintroduce BOM.